src/patch_Distributions.jl

```diff
@@ -0,0 +1,82 @@
+### Maxwell Boltzmann distribution
+# https://en.wikipedia.org/wiki/Maxwell–Boltzmann_distribution
+
+const sqrt_two_over_pi = sqrt(2 / pi)
```

Comment:
This name is longer than the formula, I'm not sure it's useful to have this constant. Also, since it's a constant it should be capitalized I think.

Reply:
Done in 41314f5. 

Here, this is more of a style thing, because it gains around 1ns for single-point evals.